### DISCUSSION TITLE
修改achievements章节的标题时报错

### DISCUSSION THREAD

**Discussion by stella-zhou** [sjtug/SJTUThesis#904] [提问]
我想把achievements这章改为仅展示发表的论文，于是在setup.tex文件中进行了如下修改：
![image](https://user-images.githubusercontent.com/65953185/227719028-9b52824d-02dc-4042-9317-6021bc599ff2.png)

结果产生了以下报错：
![image](https://user-images.githubusercontent.com/65953185/227719082-c2bd7943-d061-4c1c-9ec0-5d12ff719e05.png)
请问是什么原因导致的？

**Comment by AlexaraWu** [comment]
接口简化了一下，现在是 `achv = {XXX}`。

**Reply by note286** [reply]
我觉得可以留着

```latex
achievements  .tl_set:N = \l__sjtu_name_achv_tl ,
achievements .initial:n = { List~of~Research~Achievements },
```

但是文档里不提，这样能兼容旧的`.tex`。

**Reply by stella-zhou** [reply]
> 接口简化了一下，现在是 `achv = {XXX}`。

谢谢，问题解决了。我想再另起一个章节写做过的项目，请问应该怎么做？

**Reply by LogCreative** [reply]
临时修改一下这个键，之后再使用 `achievements` 环境。

```latex
\sjtusetup{name={achv={已完成的项目}}}
\begin{achievements}
  % ...
\end{achievements}
```

> 在 2.0 开发版中，还可以使用 `\begin{achievements}[已完成的项目]`，感兴趣可以下载 [SJTUTeX](https://github.com/sjtug/SJTUTeX) 仓库解包使用。

**Reply by AlexaraWu** [reply]
在 sjtug/SJTUTeX@7f3ad2e42c9c10420e808a604e36a7efd38bfd14 中添加了对旧接口的支持。
